uber/config.py

```diff
@@ -356,7 +356,7 @@ def _is_intstr(s):
 c.STORE_ITEM_NAMES = list(c.STORE_PRICES.keys())
 c.FEE_ITEM_NAMES = list(c.FEE_PRICES.keys())
 
-c.WRISTBAND_COLORS = defaultdict(lambda: c.DEFAULT_WRISTBAND, c.WRISTBAND_COLORS)
+c.WRISTBAND_COLORS = defaultdict(lambda: c.WRISTBAND_COLORS[c.DEFAULT_WRISTBAND], c.WRISTBAND_COLORS)
```

Comment:
Minor fix; we were returning the integer value rather than the string description here, which is not what we wanted.